src/window/window_wrapper/mod.rs

```diff
@@ -321,8 +321,15 @@ impl GlutinWindowWrapper {
                         ..
                     },
                 ..
-            } => self.handle_mouse_wheel(x as i32, y as i32),
-
+            } => self.handle_mouse_wheel(x as f32, y as f32),
+            Event::WindowEvent {
+                event:
+                    WindowEvent::MouseWheel {
+                        delta: MouseScrollDelta::PixelDelta(lpos),
```

Comment:
nitpick: I try not to have abbreviations in variable names. For example its not clear to me what l in lpos stands for. Maybe this should be logical_position?

Reply:
It does stand for logical_position, ill change it back

Reply:
Other than that, this looks great! Were you able to figure out the fast scrolling issue?

Reply:
How does it feel if instead of dividing the logical_position.y by 10 you call this function: https://doc.rust-lang.org/std/primitive.f32.html#method.signum and send that to handle_mouse_wheel?

Reply:
The issue seems to be related to macOS trackpad acceleration. It works fine for my friend on Linux. I can try playing around with two more ideas but I doubt it will fix the issue

Reply:
As far as I can tell dividing by 10 has no effect not the scroll speed, so it can be removed.

Reply:
I'm happy to merge as is because its better than nothing. Just let me know if and when you finish fiddling with it!

Reply:
@Kethku Request: squash and merge